nasa/ow_simulator
# Oceanwater 624 faults code optimization

## Linked Issues:
| Jira Ticket 🎟️   | [Oceanwater-624](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-624) |
| ----------- | ----------- |
| EPIC ⚡| [Oceanwater-551](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-551) |
| Github :octocat:  | na |


## Summary of Changes
* Condensing some old code to be cleaner and have less repetitive functions
* New class vars 
* Breaking up longer functions into smaller helper functions

## Test
* roslaunch any file
* rostopic echo [any faults messages]
* observe fault message values are the same for bitmask and also components. 
* observe fault message values still working for arm and antennae]
